README.md

```diff
@@ -22,7 +22,7 @@ limitations under the License.
 [![Documentation Status](https://readthedocs.org/projects/elyra/badge/?version=latest)](https://elyra.readthedocs.io/en/latest/?badge=latest)
 [![GitHub](https://img.shields.io/badge/issue_tracking-github-blue.svg)](https://github.com/elyra-ai/elyra/issues)
 [![Gitter](https://badges.gitter.im/elyra-ai/community.svg)](https://gitter.im/elyra-ai/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
-[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/elyra-ai/elyra/master?urlpath=lab/tree/binder-demo)
+[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/elyra-ai/elyra/v1.0.0?urlpath=lab/tree/binder-demo)
```

Comment:
I would say that we want to have the top `binder` badge should build master on the master branch. We could magically update when a release is created

Reply:
Good point. That would certainly be consistent with the documentation badge - which is tied to 'latest' (i.e., master).

Reply:
Personally I would remove that badge altogether or at least change the label to avoid any ambiguity. Since we don't know the user's intention why they are launching JL/Elyra in Binder either choice could be "invalid". That said, if you prefer building off master I can revert the change.

Reply:
I removed the link @lresende. If I misinterpreted your thumbs up please let me know 😄